.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+# test files
+/typeset/test
```

Comment:
Can these tests be included in the repo? And can `npm test` be updated to run those tests? I use https://facebook.github.io/jest/ since it detects dependencies in files, uses `git status` to determine which tests to run first, and re-runs tests when files change.

Also, I am OK with adding tests in a subsequent PullRequest if folks would rather have this merged soon.